event/src/main/java/jetbrains/jetpad/event/PropertyChangeEventWrapper.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2012-2016 JetBrains s.r.o
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package jetbrains.jetpad.event;
+
+import jetbrains.jetpad.model.property.PropertyChangeEvent;
+
+public class PropertyChangeEventWrapper extends Event {
+  public final PropertyChangeEvent<?> event;
```

Comment:
It's not code policy. We definitely have a lot of public final fields in the code.
I just think that in this particular case (public class, open-source repository) we could be more defensive against possible future refactorings.